server/services/datacommons.py

```diff
@@ -336,15 +336,10 @@ def resolve(nodes, prop):
   return post(url, {'nodes': nodes, 'property': prop})
 
 
-def nl_search_vars(queries,
-                   index_types: List[str],
-                   skip_topics=False,
-                   reranker=''):
+def nl_search_vars(queries, index_types: List[str], reranker=''):
   """Search sv from NL server."""
   idx_params = ','.join(index_types)
   url = f'{current_app.config["NL_ROOT"]}/api/search_vars?idx={idx_params}'
-  if skip_topics:
-    url = f'{url}&skip_topics={skip_topics}'
```

Comment:
TODO:  cleanup skip_topics argument in nl_server?

Reply:
done